### PR TITLE
Homepage Info Section

### PR DESCRIPTION
**Pull request by sdmungov**
added the first draft of the info section of the home page and is fairly responsive

### CODE REVIEW

File: components/infoColumns/index.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  ${breakpoints("margin", "", [{ 1400: "1rem 0" }])};
+`;
+const Title = styled(SubTitle)`
+  font-size: 4rem;
+  ${breakpoints("font-size", "", [{ 1000: "2.5rem" }])};
+  width: 100px;
+  ${breakpoints("width", "", [{ 1000: "100%" }])};
+  margin: 0;
+`;
+
```
Comment (he-james) [inline components/infoColumns/index.tsx]: I would move these to its own file, maybe

File: components/infoColumns/index.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+          of their nation's stance and role on the global stage, rather than for
+          passing empty resolutions or for compromising their nation's values.
+          <br />
+          <br />
+          As an independent conference, CIMUN is not affiliated with any
+          university. Rather, we are a diverse and talented team of students,
+          post-graduates, and professionals from backgrounds in academia,
+          finance, politics, business, technology and more to deliver a
+          high-caliber educational experience to our delegates.
+        </Body>
```
Comment (he-james) [inline components/infoColumns/index.tsx]: I don't really mind this, but I personally think large blocks of text should be mapped onto an object in another file for convenient editing

File: pages/index.tsx [hunk 3]
```diff
@@ -15,59 +17,38 @@ export default function Home() {
...
-      </ComponentWrapper>
+      <InfoColumns />
       <footer className={styles.footer}>
         <a
           href="https://vercel.com?utm_source=model-un&utm_campaign=oss"
           target="_blank"
           rel="noopener noreferrer"
         >
-          Powered by{' '}
+          Powered by{" "}
```
Comment (he-james) [inline pages/index.tsx:46]: hahaha linter

File: styles/typography.ts [hunk 4]
```diff
@@ -51,4 +54,40 @@ export const Header = styled.h2<TextType>`
       props.line ? props.line : undefined};
   width: ${(props: TextType) =>
       props.width ? props.width : undefined};
+  text-align: ${(props: TextType) =>
+      props.align ? props.align : 'left'};
+`;
+
+export const SubTitle = styled.h3<TextType>`
```
Comment (he-james) [inline styles/typography.ts:64]: i can see that you are becoming a styled component god

**Review by he-james — APPROVED** [review]

**Review by FB-Reda — APPROVED** [review]

**Pull request merged**